.drone.yml

```diff
@@ -1,40 +1,33 @@
-build:
-  image: golang:1.5
-  environment:
-    - CGO_ENABLED=0
-  commands:
-    - make deps
-    - make vet
-    - make build
-    - make test
+workspace:
+  base: /go
+  path: src/github.com/drone-plugins/drone-google-cloudstorage
+
+pipeline:
+  build:
+    image: golang:1.9
+    environment:
+      - CGO_ENABLED=0
+    commands:
+      - make deps
+      - make vet
+      - make test
+      - make build
 
-publish:
-  coverage:
-    when:
-      branch: master
   docker:
-    username: $$DOCKER_USER
-    password: $$DOCKER_PASS
-    email: $$DOCKER_EMAIL
-    repo: plugins/drone-gcs
+    image: plugins/docker
+    pull: true
+    secrets: [ docker_username, docker_password ]
+    repo: plugins/gcs
     tag: latest
```

Comment:
I wasn't sure what the versioning should look like. Let me know and I can update to whatever works best.

Reply:
You could look at https://github.com/drone-plugins/drone-google-cloudstorage/blob/7ca304836e70dbd1e09dd23b83a3886b0753d43b/.drone.yml (which comes from #11), this should be more or less the standard drone config for repos within this org.

Reply:
Thanks for pointing me in the right direction. 

Can we get the #11 merged and I'll rebase this with updated master? I would prefer not to expand the scope of this PR. 

Also, if this is a new standard to be followed, can the https://github.com/msuterski/drone-plugin-starter be updated? It's very outdated.

Reply:
> Also, if this is a new standard to be followed, can the https://github.com/msuterski/drone-plugin-starter be updated? It's very outdated.

Yeah, this should also get updated :)